### ISSUE TITLE
Suggested fix for deprecated 'from sklearn.utils.linear_assignment_ import linear_assignment'

### ISSUE THREAD

**Issue by AliS567** [XifengGuo/DEC-keras#25]
Hello great work! 

i think 'from sklearn.utils.linear_assignment_ import linear_assignment' is now deprecated and I would recommend making the following changes to the accuracy module.

def acc(y_true, y_pred):
    """
    Calculate clustering accuracy. Require scikit-learn installed
    # Arguments
        y: true labels, numpy.array with shape `(n_samples,)`
        y_pred: predicted labels, numpy.array with shape `(n_samples,)`
    # Return
        accuracy, in [0,1]
    """
    y_true = y_true.astype(np.int64)
    assert y_pred.size == y_true.size
    D = max(y_pred.max(), y_true.max()) + 1
    w = np.zeros((D, D), dtype=np.int64)
    for i in range(y_pred.size):
        w[y_pred[i], y_true[i]] += 1
    from scipy.optimize import linear_sum_assignment as linear_assignment
    ind = np.transpose(np.asarray(linear_assignment(w.max() - w)))
    return sum([w[i, j] for i, j in ind]) * 1.0 / y_pred.size

Thanks for all the great work!
Ali

**Comment by enaserianhanzaei** [comment]
I just found the sklearn linear-assignment and added that to the code (locally)
